app/dataaccess/base/MongoConnector.py

```diff
@@ -1,29 +1,40 @@
-from logging import Logger
+import logging
 
+from dependency_injector import containers, providers
 from pymongo import MongoClient
 from pymongo.errors import ConnectionFailure
 
 from app.utils.configuration import MASConfiguration
+from app.utils.log_config import LogConfig
 
 
 class MongoConnector:
-    def __init__(self, db_name: str, logger: Logger):
-        self.logger = logger
-        self.db_name = db_name
+    def __init__(self):
         self._client = None
-        self.db = None
         self._connect()
+        self.logger = None
```

Comment:
Czy tu powinno być to logger=None? Bo ta metoda _connect() go inicjalizuje

Reply:
myślałem że vsc będzie mi krzyczeć, że nie inicjuje loggera w konstruktorze tylko w metodzie `connect`, ale racja zmienię